styles/proton/catppuccin.user.css

```diff
@@ -54,6 +67,7 @@
     @mantle: @catppuccin[@@lookup][@mantle];
     @crust: @catppuccin[@@lookup][@crust];
     @accent-Color: @catppuccin[@@lookup][@@accent];
+    @flavor: @lookup;
```

Comment:
I think its better to remove this line and replace later instances of `@flavor` with `@lookup`